airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/spec.json

```diff
@@ -67,14 +51,30 @@
               },
               "access_token": {
                 "type": "string",
-                "title": "Access Token",
+                "title": "Access token",
                 "description": "The token value generated using the authentication code. See the <a href=\"https://docs.airbyte.com/integrations/sources/linkedin-ads#authentication\">docs</a> to obtain yours.",
                 "airbyte_secret": true
               }
             }
           }
         ]
-      }
+      },
+      "start_date": {
```

Comment:
did you want to change the order of these? you'll need to add the `order` property